src/validation.cpp

```diff
@@ -5644,6 +5659,7 @@ Chainstate& ChainstateManager::ActivateExistingSnapshot(CTxMemPool* mempool, uin
         std::make_unique<Chainstate>(mempool, m_blockman, *this, base_blockhash);
     LogPrintf("[snapshot] switching active chainstate to %s\n", m_snapshot_chainstate->ToString());
     m_active_chainstate = m_snapshot_chainstate.get();
+    m_ibd_chainstate->m_snapshot_entry = m_snapshot_chainstate->m_snapshot_entry;
```

Comment:
I don't think this line does anything. It's called during init from `LoadChainstate() -> DetectSnapshotChainstate()`, at which point `m_snapshot_chainstate->m_snapshot_entry` isn't set yet. That only happens right after, in `CompleteChainstateInitialization() -> LoadBlockIndex()`

Reply:
Good catch, removed this and left a comment there instead.